libkbfs/config_local.go

```diff
@@ -897,9 +898,16 @@ func (c *ConfigLocal) EnableJournaling(
 	log := c.MakeLogger("")
 	branchListener := c.KBFSOps().(branchChangeListener)
 	flushListener := c.KBFSOps().(mdFlushListener)
+	// Set the journal disk limit to 10 GiB for now. TODO: Make
+	// this configurable.
+	var journalDiskLimit int64 = 10 * 1024 * 1024 * 1024
```

Comment:
I'm not sure about this default.  It might be better to do something based on the overall size of the disk (e.g., with https://github.com/ricochet2200/go-disk-usage)?  A strawman proposal: a quarter of the total size of the disk, up to a max of 100 GB maybe?

Ideally it would somehow scale dynamically based on the amount of available space not being used by the journal, but that would be tough to implement.

I think the first release should give a decent experience out of the box for both those with small and big disks, whenever possible.